packages/fiori/src/Timeline.js

```diff
@@ -2,10 +2,10 @@ import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
 import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
 import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
 import { isTabNext, isTabPrevious } from "@ui5/webcomponents-base/dist/Keys.js";
+import { getEffectiveAriaLabelText } from "@ui5/webcomponents-base/dist/util/AriaLabelHelper.js";
```

Comment:
The point of using the getEffectiveAriaLabelText method is to return the aria accessible name by checking the accessibleName and accessibleNameRef properties, but the Timeline has only accessibleName. It doesn't harm, it won't break the code, but it's not need it and performs unneeded checks.